packages/create-vite/src/index.ts

```diff
@@ -246,6 +246,41 @@ const FRAMEWORKS: Framework[] = [
       },
     ],
   },
+  {
+    name: 'ember',
+    display: 'Ember',
+    color: redBright,
+    variants: [
+      {
+        name: 'ember app',
+        display: 'Production-ready App',
+        color: redBright,
+        customCommand:
+          'npx ember-cli@latest new TARGET_DIR --blueprint @ember/app-blueprint --pnpm --lang en',
+      },
+      {
+        name: 'ember app (ts)',
+        display: 'Production-ready App (TypeScript)',
+        color: blueBright,
+        customCommand:
+          'npx ember-cli@latest new TARGET_DIR --blueprint @ember/app-blueprint --pnpm --lang en --typescript',
+      },
+      {
+        name: 'ember library',
+        display: 'Production and OSS ready Library',
+        color: redBright,
+        customCommand:
+          'npx ember-cli@latest addon TARGET_DIR --blueprint @ember/addon-blueprint --pnpm --typescript',
```

Comment:
do ember libraries leverage vite during their build process? (vite library mode?)  create-vite is primarily used for creating applications using vite, i don't think we have explicit library support for any other option